diameter_synthesis/__init__.py

```diff
@@ -5,18 +5,7 @@
 
 # Copyright (C) 2021  Blue Brain Project, EPFL
```

Comment:
Shouldn't this be an interval 2021-2024?

Reply:
Seems so indeed